kong/tracing/propagation.lua

```diff
@@ -479,7 +483,7 @@ local function set(conf_header_type, found_header_type, proxy_span, conf_default
   end
 
   if conf_header_type == "ot" or found_header_type == "ot" then
-    set_header("ot-tracer-traceid", to_hex(proxy_span.trace_id))
+    set_header("ot-tracer-traceid", to_hex(to_ot_trace_id(proxy_span.trace_id)))
```

Comment:
The only specification I could find about the format of the `ot-tracer-traceid` header is [this one](https://www.npmjs.com/package/@opentelemetry/propagator-ot-trace/v/0.25.0), where  `ot-tracer-traceid` and `ot-tracer-spanid` are both defined to be the same size: _a string of 16 hex characters_ (while here we are using 32). Am I missing something?